### DISCUSSION TITLE
pyarmor license

### DISCUSSION THREAD

**Discussion by SirKnightV** [dashingsoft/pyarmor#1309] [Q&A] [answered]
Hello, i have updated my pyarmor license and read the doc about the changes, but i dont understand something yet, the pyarmor license .zip we can use it unlimited times? or is limited to 10 uses max?, and my other question is, why is pyarmor reg code .txt limited max to 10 times and what is the difference about the .zip ?

**Comment by jondy** [comment]
The pyarmor license .zip can be used unlimited times.

**Reply by SirKnightV** [reply]
> The pyarmor license .zip can be used unlimited times.

hello jondy, but my question is, why the .txt can be used only 10 times and why the .zip is unlimited, what is the difference?

**Comment by jondy** [comment, marked as answer]
The .txt is designed to activate this license,  generally use it only once. It will request remote server to generate .zip file, the .zip includes real registration file. Once got .zip file, it's not necessary to request server once more.
